game/game.go

```diff
@@ -43,6 +43,9 @@ type Game struct {
 
 	// Permanents contains all permanents in play.
 	Permanents map[PermanentId]*Permanent
+
+	// Non-mana, non-cost effect actions go on the stack and can be responded to before they resolve
+	Stack []*Action
```

Comment:
Hmm, the data format gets messy again if Game points to Action. Can we keep the Action object so that it only represents actions taken by players? Things can go on the stack when they do not have an acting player, like triggered abilities.